### ISSUE TITLE
Add a status change for when button pressed but no power state change?

### ISSUE THREAD

**Issue by hepcat72** [tabrindle/homebridge-leviton#35]
I just got off the phone with Leviton and discussed with them what I want to do:

I control the state of 2 other switches (my driveway flood light and my garage eaves lights) with my Leviton dimmer switch (the backyard flood light).  The problem is that those other 2 switches can also be controlled independently and those switches are located far from the leviton switch (each on a separate floor), and if say, the driveway flood is on and I want to turn it off, pressing the off button on the leviton switch will not do it. That's because I have to poll the leviton for its state and the json that's sent only has power state and brightness.  There's nothing to indicate if the button was pressed if it does not change the power state or brightness.  I.e. if the backyard flood is off and I press the off button, there's not state change to catch.

One other tidbit that may be important is that I use the homebridge automation node in node red to effect the automations.  My polling uses the `hb-status` node.  My understanding is that the `hb-event` node is supposed to emit events, but I'm guessing that it is only status change events (power state and brightness).

There are 2 ways this problem could be solved. Either the `hb-event` node could emit an event when the button is pressed even though there's not state change in the json it sends compared to what I see from polling or an extra state(s) could be added to the json (e.g. {"off-button": true/false}), similar to the way my wemo motion device changes {"On": flase} to {"On: true} for a few seconds when it detects motion.

I don't actually know if the content of the json is dictated by the homebridge automation node or if it is determined by this homebridge plugin, but the guy with Leviton claimed that they have IFTTT integrations that can be set up to trigger actions by the on/off buttons regardless of the power state, so there must be some sort of API to communicate those button presses.

I don't know if this is a feature that homebridge-leviton can solve.  I imagine the data communicated to homebridge is hemmed in by what the homekit protocols dictate?  And my guess would be that there does not exist a piece of data that can be included to indicate when a button is pressed that doesn't change the power state?  Can you provide any insights?

**Comment by hepcat72** [comment]
I also tried to find API documentation at Leviton and came up dry...

**Comment by hepcat72** [comment]
I explored Leviton's IFTTT channel and there does appear to be a button pressed trigger to choose from, so I tried reaching out to Leviton support via email and got a pretty detailed answer.  They have 1 device where button press events are a separate status from the power state or anything else, but that device is not my dimmer.  I submitted a feature request, but given that this doesn't appear to be possible, I am closing this issue.
